### ISSUE TITLE
Lua Debugger?

### ISSUE THREAD

**Issue by SafeteeWoW** [evil-morfar/RCLootCouncil2#167]
Do you know any good editor that can debug Lua 5.1?
I have tried a lot, and the only one I have found that actually works is LuaStudio, but it costs $35.

**Comment by evil-morfar** [comment]
No, but then again I've never really had a need for it. Atom + [Script](https://atom.io/packages/script) has been enough for me for a long time.
